snmp/metadata.csv

```diff
@@ -701,6 +702,9 @@ snmp.ltmVirtualServStatTotRequests,count,,,,[F5 BIG-IP] The total number of requ
 snmp.ltmVirtualServStatVsUsageRatio1m,gauge,,,,[F5 BIG-IP] The percentage of time a given virtual server was busy over the last minute.,0,snmp,,
 snmp.ltmVirtualServStatVsUsageRatio5m,gauge,,,,[F5 BIG-IP] The percentage of time a given virtual server was busy over the last 5 minutes.,0,snmp,,
 snmp.ltmVirtualServStatVsUsageRatio5s,gauge,,,,[F5 BIG-IP] The percentage of time a given virtual server was busy over the last 5 seconds.,0,snmp,,
+snmp.ltmVsStatus,gauge,,,,[F5 BIG-IP] Stores status information of virtual servers.,0,snmp,,
+snmp.ltmVsStatusEnabledState,gauge,,,,[F5 BIG-IP] The activity status of the specified virtual server (as specified by the user).,0,snmp,,
+snmp.ltmVsStatusName,gauge,,,,[F5 BIG-IP] The name of a virtual server.,0,snmp,,
```

Comment:
```suggestion
```

Reply:
👋 Those two are tags, not metrics, we don't expected them to be present in metadata.csv.